### PR TITLE
Allow sharing of JmDNS among multiple HomekitServer instances

### PR DESCRIPTION
**Pull request by ccutrer**
for niceness when having multiple bridges in a single process

### CODE REVIEW

**Comment by ccutrer** [comment]
extra bonus: in OpenHAB, we'll be able to use the JmDNS that ESH already manages

File: pom.xml [hunk 1]
```diff
@@ -87,9 +87,9 @@
 		</dependency>
 
 		<dependency>
-			<groupId>javax.jmdns</groupId>
+			<groupId>org.jmdns</groupId>
 			<artifactId>jmdns</artifactId>
-			<version>3.4.1</version>
```
Comment (timcharper) [inline pom.xml]: hmm... breaking change to bump a minor version of a dependency? May need to wait for 1.2.0 release?

Comment (rhwood) [reply pom.xml]: Why is changing the maven organization from the abandoned name to the maintained name a breaking change (the implemented API did not change in a breaking way)?

**Comment by ccutrer** [comment]
Superseded by #122

**Pull request closed without merging**